src/content/en/updates/2017/02/media-session.md

```diff
@@ -0,0 +1,398 @@
+  .catch(error => { console.log(error) });
+});
+
+</pre>
+
+### Customize the notification
+
+When your web app is playing audio, you can already see a media notification
+sitting in the notification tray. On Android, Chrome does its best there to show
+appropriate information by using the document's title and the largest favicon
+it can find.
+
+<div class="clearfix"></div>
+<div class="attempt-left">
+  <figure>
+    <img src="/web/updates/images/2017/02/without-media-session.png" alt="Without Media Session">
+    <figcaption>With no Media Session</figcaption>
+  </figure>
+</div>
+<div class="attempt-right">
+  <figure>
+    <img src="/web/updates/images/2017/02/with-media-session.png" alt="With Media Session">
+    <figcaption>With Media Session</figcaption>
+  </figure>
+</div>
+
+Let's see how to customize this media notification by setting some media
+session metadata such as the title, artist, album name, and artwork with the
+Media Session API.
+
+    if ('mediaSession' in navigator) {
+
+      navigator.mediaSession.metadata = new MediaMetadata({
+        title: 'Never Gonna Give You Up',
+        artist: 'Rick Astley',
+        album: 'Whenever You Need Somebody',
+        artwork: [
+          { src: 'https://dummyimage.com/96x96',   sizes: '96x96',   type: 'image/png' },
+          { src: 'https://dummyimage.com/128x128', sizes: '128x128', type: 'image/png' },
+          { src: 'https://dummyimage.com/192x192', sizes: '192x192', type: 'image/png' },
+          { src: 'https://dummyimage.com/256x256', sizes: '256x256', type: 'image/png' },
+          { src: 'https://dummyimage.com/384x384', sizes: '384x384', type: 'image/png' },
+          { src: 'https://dummyimage.com/512x512', sizes: '512x512', type: 'image/png' },
+        ]
+      });
+    }
+
+If your web app provides a playlist for instance, you may want to allow the
+user to navigate through your playlist directly from the media notification
+with some "Previous Track" and "Next Track" icons.
+
+    let audio = document.createElement('audio');
+
+    let playlist = ['audio1.mp3', 'audio2.mp3', 'audio3.mp3'];
+    let index = 0;
+    
+    navigator.mediaSession.setActionHandler('previoustrack', function() {
+      // User clicked "Previous Track" media notification icon.
+      index = (index - 1 + playlist.length) % playlist.length;
+      playAudio();
+    });
+    
+    navigator.mediaSession.setActionHandler('nexttrack', function() {
+      // User clicked "Next Track" media notification icon.
+      index = (index + 1) % playlist.length;
+      playAudio();
+    });
+
+    function playAudio() {
+      audio.src = playlist[index];
+      audio.play()
+      .then(_ => { /* Set up Media Session... */ })
+      .catch(error => { console.log(error); });
+    }
+
+    playButton.addEventListener('pointerup', function(event) {
+      playAudio();
+    });
+
+The Media Session API allows you as well to show "Seek Backward" and "Seek
```

Comment:
nit: Remove "as well"

It's kinda suggested here, but is it worth being explicit that these buttons won't be shown unless you add a handler?